service/src/main/java/bio/terra/landingzone/service/landingzone/azure/model/LandingZoneResourcesByPurpose.java

```diff
@@ -0,0 +1,40 @@
+package bio.terra.landingzone.service.landingzone.azure.model;
+
+import java.util.List;
+import java.util.Map;
+
+public record LandingZoneResourcesByPurpose(
+    Map<String, List<LandingZoneResource>> generalResources,
+    Map<String, List<LandingZoneSubnetResource>> subnetResources) {
+
+  public static Builder builder() {
```

Comment:
Wondering if we need the Builder pattern for a record with 2 fields..

Reply:
Oh, thanks - was changing to and forth, did not notice there are only 2 fields left.